app/errors.py

```diff
@@ -0,0 +1,14 @@
+class VaccineError(Exception):
+    pass

```

Comment:
You should add a custom error message to your `VaccineError` class. It's a good practice to provide descriptive error messages that can help other developers understand what went wrong when the exception is raised.